client/extensions/woocommerce/app/products/product-form-variations-table.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { isNumber } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ProductFormVariationRow from './product-form-variation-row';
+
+class ProductFormVariationsTable extends React.Component {
+
+	static propTypes = {
+		variations: PropTypes.array,
+		product: PropTypes.object,
+		editProductVariation: PropTypes.func.isRequired,
+	};
+
+	renderVariationRow( variation ) {
+		const { product, variations, editProductVariation } = this.props;
+		const id = isNumber( variation.id ) && variation.id || 'index_' + variation.id.index;
+		const manageStock = variations && variations[ 0 ].manage_stock || false;
```

Comment:
No worries! Thanks for providing some feedback already. I'll be working on splitting this branch up into smaller PRs tomorrow and will doing some cleanup using your notes.